README.md

```diff
@@ -17,7 +15,35 @@ Orkestra is built on top of popular [CNCF](https://cncf.io/) tools and technolog
 - [Chartmuseum](https://chartmuseum.com/)
 - [Keptn](https://keptn.sh)
 
-<p align="center"><img src="./docs/assets/orkestra-core.png" width="750x" /></p>
+![Orkestra Core](docs/assets/orkestra-core.png)
+
+## Table of Contents
+
+- [Orkestra](#orkestra)
+  - [Background and Motivation](#background-and-motivation)
```

Comment:
Can we switch the `Background and Motivation` sections with the `What is Orkestra?` section? We could also consider putting it under `Overview` if we grouped these all together